solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/AccountTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  AccountTests.swift
+//  FinanceAppTests
+//
+//  Created by Gabriel de Castro Chaves on 23/08/22.
+//
+
+import XCTest
+@testable import FinanceApp
+
+class AccountTests: XCTestCase {
```

Comment:
[suggestion] uma boa pratica e colocar `final`:
```suggestion
final class AccountTests: XCTestCase {
```